### PR TITLE
imp: S3UTILS-101 improve stalled retry query

### PR DESCRIPTION
**Pull request by naren-rajendran**
Remove regex filter on id and handle master key check on the client.

### CODE REVIEW

File: stalledRetry.yaml [hunk 1]
```diff
@@ -53,3 +53,7 @@ spec:
               # path to write heap snapshots
             - name: HEAP_PROFILER_PATH
               value: "/path/to/write"
+              # https://www.mongodb.com/docs/v4.2/core/read-preference/index.html
+              # mongodb read preference - primary|secondary|primaryPreferred|secondaryPreferred|nearest
+            - name: MONGODB_READ_PREFERENCE
+              value: "primary"
```
Comment (rahulreddy) [inline stalledRetry.yaml]: Is there any reason you are targeting Primary?

**Review by rahulreddy — CHANGES_REQUESTED** [review]

File: stalledRetry.yaml (hunk 1 above)
Comment (naren-rajendran) [reply stalledRetry.yaml]: It was already forced to use `primary` https://github.com/scality/s3utils/pull/258/files#diff-9b354bcb233267c1a41c1bd8dd4cc82ddef2ae458fed06e07697237a420e98e4L85 that is why I wanted to have this as a variable.

Comment (naren-rajendran) [reply stalledRetry.yaml]: This is only with stalled (not sure why), count items uses `secondaryPreferred`.

Comment (rahulreddy) [reply stalledRetry.yaml]: Ok unless writes are needed you should go with secondary preferred

Comment (naren-rajendran) [reply stalledRetry.yaml]: Modified.

**Review by rahulreddy — APPROVED** [review]

**Review by miniscruff — APPROVED** [review]

**Pull request merged**